src/worker/sandbox/cgroups/pool.go

```diff
@@ -59,30 +60,24 @@ func (pool *CgroupPool) NewCgroup() Cgroup {
 	cg := &CgroupImpl{
 		name: fmt.Sprintf("cg-%d", pool.nextID),
 		pool: pool,
+		log:  pool.log.With("cg", fmt.Sprintf("cg-%d", pool.nextID)),
```

Comment:
Let's avoid copying code for the cg name in case we choose a different naming scheme later.  Better to store the name in a separate variable, then use it in both places.